docs/tutorials/notebooks/basic_ot_between_datasets.ipynb

```diff
@@ -0,0 +1,376 @@
+{
```

Comment:
<p>It could be nice to also plot the components of this gradient on top of the data in the scatterplot above as another way of illustrating how it pushes the measures closer together</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>good idea :) I added this.</p>